src/js/index.js

```diff
@@ -1 +1,34 @@
 import "../styles/index.css";
+import { startingPage } from "./starting-page";
+import { firstChapter, secondChapter, thirdChapter } from "./demo-story";
+import { createChapterStructure } from "./chapterDOM";
+import { render } from "./render";
+
+//Now it's just one story object, later we should create factory for stories
+const storyObj = {
+  chapters: [],
+  addChapter(...chapterData) {
+    chapterData.forEach((data) => {
+      this.chapters.push(data);
+    });
+  },
+
+  findNextChapter(currentChapter) {
+    const chapterNumber = currentChapter.getChapterNumber();
+
+    return this.chapters.find(
+      (nextChapter) => nextChapter.getChapterNumber() === chapterNumber + 1
+    );
+  },
+};
+
+storyObj.addChapter(firstChapter, thirdChapter, secondChapter);
+
+const firstChapterStructure = createChapterStructure(firstChapter);
+const startPage = startingPage(() =>
+  render(firstChapterStructure.setupChapterPage())
```

Comment:
Would be nice if setupChapterPage() was already included in createChapterStructure(firstChapter). Then it is a bit less code and easier to read imo. 

You can do this by returning setupchapterPage() in chapterDOM.js instead of { setupChapterPage }

Reply:
To be honest i didn't even tought that i can do it this way. It's very nice addition for more readable code. Thanks for the info and of course i will implement this solution!